doc/api/http.md

```diff
@@ -2426,6 +2426,13 @@ as an argument to any listeners on the event.
 <!-- YAML
 added: v0.1.5
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/45982
```

Comment:
Do you need to update this PR URL to be 46240 or add it as a second pr-url entry? Or do we not do that for backports?

Reply:
We don't do that for backports. Example https://github.com/nodejs/node/pull/42298/files.